### PR TITLE
Update sde_solve.md

### PR DESCRIPTION
**Pull request by onoderat**
Included the documentation for the PCEuler algorithm. Made a slight edit in the equation for commutativity.

### CODE REVIEW

File: docs/src/solvers/sde_solve.md [hunk 1]
```diff
@@ -64,6 +64,13 @@ Orders are given in terms of strong order.
   all forms of noise, including non-diagonal, scalar, and colored noise.†
 - `EulerHeun` - The Euler-Heun method. Strong Order 0.5 in the Stratonovich sense.
   Can handle all forms of noise, including non-diagonal, scalar, and colored noise.†
+- `PCEuler` - The predictor corrector euler method. Strong Order 0.5 in the Ito
+  sense. Requires the bbprime function, which is defined as
+  ```math
```
Comment (ChrisRackauckas) [inline docs/src/solvers/sde_solve.md]: Move this above to a section, like commutative noise.

File: docs/src/solvers/sde_solve.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ a matrix in the `SDEProblem` type. A special form of non-diagonal noise,
 commutative noise, occurs when the noise satisfies the following condition:
 
 ```math
-\sum_{i=1}^d g_{i,j_1}(t,u) \frac{\partial g_{k,j_2}}{\partial x_i} = \sum_{i=1}^d g_{i,j_2}(t,x) \frac{\partial g_{k,j_1}}{\partial x_i}
+\sum_{i=1}^d g_{i,j_1}(t,x) \frac{\partial g_{k,j_2}(t,x)}{\partial x_i} = \sum_{i=1}^d g_{i,j_2}(t,x) \frac{\partial g_{k,j_1}(t,x)}{\partial x_i}
```
Comment (ChrisRackauckas) [inline docs/src/solvers/sde_solve.md]: BTW, did you ask whether there's a nice way to write this for `g` a matrix?

Comment (onoderat) [reply docs/src/solvers/sde_solve.md]: I decided to keep my email short (to maximize reply probability) so I did not include that. But yeah, I double checked a couple of times and am pretty sure that the form that you came up with is correct.

File: docs/src/solvers/sde_solve.md (hunk 1 above)
Comment (onoderat) [reply docs/src/solvers/sde_solve.md]: Made a new "Analytic Methods" section. Presumably, things like the Mil with analytic derivatives will fall under this method as well. What do you think?

File: docs/src/solvers/sde_solve.md [hunk 3]
```diff
@@ -115,6 +115,17 @@ For `SRA` and `SRI`, the following option is allowed:
   implicit Midpoint method on the drift term and is symplectic in distribution.
   Handles diagonal and scalar noise.
 
+### Analytic Methods
```
Comment (ChrisRackauckas) [inline docs/src/solvers/sde_solve.md]: Derivative-Based Methods? Analytic methods sounds undescriptive to me.

File: docs/src/solvers/sde_solve.md (hunk 2 above)
Comment (ChrisRackauckas) [reply docs/src/solvers/sde_solve.md]: Can we add the matrix way of writing the conditions while we're at it?

**Pull request merged**